js/app.js

```diff
@@ -145,7 +152,8 @@ angular.module('PathOfDamage', [])
     var section = $scope.sections[sectionKey];
     Object.keys(section.tables).forEach(function (tableKey) {
       var table = section.tables[tableKey];
-      table.totalCalc = table.totalCalc || DataService.additive;
+      table.calcTotal = table.calcTotal || DataService.additive;
```

Comment:
yeah, will probably go away with prototypes refactor, but it's an easy change for now.